events/code_review_events/workflow.py

```diff
@@ -219,7 +225,44 @@ def publish_results(self, payload):
         return True
 
     def parse_pulse(self, payload):
-        assert self.publish is True, "Publication disabled"
+
+        routing = payload["routing"]
+
+        # Process autoland payloads
+        if routing["exchange"] == PULSE_TASK_GROUP_RESOLVED:
+            try:
+                self.trigger_autoland(payload["body"])
+            except Exception as e:
+                logger.warn(
+                    "Autoland trigger failure", key=routing["key"], error=str(e)
+                )
+        else:
+            logger.info("Skipping pulse message", key=routing["key"])
+
+    def trigger_autoland(self, body: dict):
+        """
+        Trigger a code review autoland ingestion task
+        If the task is an autoland decision task
+        """
+        # Load first task in task group, check if it's an autoland
+        queue = taskcluster_config.get_service("queue")
+        task_group_id = body["taskGroupId"]
+        logger.info("Checking autoland task", task_group_id=task_group_id)
+        task = queue.task(task_group_id)
```

Comment:
Could we check directly from the pulse message data rather than making a request to load the task?

Reply:
Unfortunately no, the task group payload has just the group id, no routes, nothing (which makes sense as a group is just an id used by many tasks)

Reply:
(The info is likely at least in "other_routes")

Reply:
That's a pity :(